src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/readthreading/ReadThreadingAssembler.java

```diff
@@ -94,20 +93,18 @@ public ReadThreadingAssembler(final int maxAllowedPathsForReadThreadingAssembler
 
     /**
      * Main entry point into the assembly engine. Build a set of deBruijn graphs out of the provided reference sequence and list of reads
-     * @param aligner
      * @param assemblyRegion              AssemblyRegion object holding the reads which are to be used during assembly
      * @param refHaplotype              reference haplotype object
      * @param fullReferenceWithPadding  byte array holding the reference sequence with padding
      * @param refLoc                    GenomeLoc object corresponding to the reference sequence with padding
-     * @param givenAlleles   the alleles to inject into the haplotypes during GGA mode
      * @param readErrorCorrector        a ReadErrorCorrector object, if read are to be corrected before assembly. Can be null if no error corrector is to be used.
+     * @param aligner
```

Comment:
If you're going to modify the java doc anyway, can you add a blurb for the aligner?

Reply:
done